clean_code_main/clean_code_checks/y_check_chain_decl_usage.clas.abap

```diff
@@ -20,12 +19,12 @@ CLASS Y_CHECK_CHAIN_DECL_USAGE IMPLEMENTATION.
 
     description = 'Chain Declarations Usage'(001).
     category    = 'Y_CHECK_CATEGORY'.
-    position    = '850'.
+    position    = '050'.
     version     = '0000'.
     has_documentation = abap_true.
 
     settings-pseudo_comment = '"#EC CHAIN_DECL_USAG' ##NO_TEXT.
-    settings-disable_threshold_selection = abap_true.
+    settings-disable_threshold_selection = abap_false.
```

Comment:
why it's false?

Reply:
My bad. It must be disabled since there is no threshold for chain declaration usages (you use, or you don't).